x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/basic_license_essentials_tier/export_rules.ts

```diff
@@ -63,6 +65,24 @@ export default ({ getService }: FtrProviderContext): void => {
         expect(exportedRule).toMatchObject(ruleToExport);
       });
 
+      it('should export defaultable fields when values are set', async () => {
+        const defaultableFields: BaseDefaultableFields = {
+          setup: '# some setup markdown',
+        };
+        const ruleToExport = getCustomQueryRuleParams(defaultableFields);
+
+        await securitySolutionApi.createRule({ body: ruleToExport });
+
+        const { body } = await securitySolutionApi
```

Comment:
Nit: either change the rest of this file to use `securitySolutionApi`, or invoke the API in the same way that the other tests are.